app/views/sign_up/passwords/new.html.erb

```diff
@@ -7,15 +7,14 @@
 </p>
 <%= simple_form_for(
       @password_form,
+      as: :password_form,
```

Comment:
What does the `as` do?

Reply:
This was necessary to keep the name in params constant when there was inheritance.